lib/src/code_field/code_controller.dart

```diff
@@ -96,6 +98,8 @@ class CodeController extends TextEditingController {
     CopySelectionTextIntent: CopyAction(controller: this),
     RedoTextIntent: RedoAction(controller: this),
     UndoTextIntent: UndoAction(controller: this),
+    TabIntent: TabIntentAction(controller: this),
+    UntabIntent: UntabIntentAction(controller: this),
```

Comment:
https://medium.com/flutter-senior/sort-alphabetically-everything-you-can-1e9d7299c4ae